Pipeline failing due to 404 error on download.

Error Logs

```
Ign:2 https://security.ubuntu.com/ubuntu noble-updates/main amd64 libxml2-utils amd64 2.9.14+dfsg-1.3ubuntu3.4
Err:2 mirror+file:/etc/apt/apt-mirrors.txt noble-updates/main amd64 libxml2-utils amd64 2.9.14+dfsg-1.3ubuntu3.4
  404  Not Found [IP: 20.106.104.242 80]
E: Failed to fetch mirror+file:/etc/apt/apt-mirrors.txt/pool/main/libx/libxml2/libxml2-utils_2.9.14%2bdfsg-1.3ubuntu3.4_amd64.deb  404  Not Found [IP: 20.106.104.242 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
Error: Process completed with exit code 100.
```

--- a/.github/workflows/pom-upgrade.yml
+++ b/.github/workflows/pom-upgrade.yml
@@ -14,12 +14,15 @@ on:
 jobs:
   check-version-upgrade:
     runs-on: ubuntu-latest
-    permissions:
-      pull-requests: write
-      contents: read
     steps:
       - name: Checkout Code
         uses: actions/checkout@v4
+
+      - name: Reset APT Sources
+        run: |
+          sudo rm -f /etc/apt/sources.list.d/*
+          sudo sed -i 's|mirror+file:/etc/apt/apt-mirrors.txt|http://archive.ubuntu.com/ubuntu|g' /etc/apt/sources.list
+          sudo apt-get update
 
       - name: Install xmllint
         run: sudo apt-get install -y libxml2-utils

